vck-aries/src/commonTest/kotlin/at/asitplus/wallet/lib/aries/IssueCredentialMessengerTest.kt

```diff
@@ -28,8 +26,8 @@ class IssueCredentialMessengerTest : FreeSpec() {
 
     init {
         beforeEach {
-            issuerKeyPair = RandomKeyPairAdapter()
-            holderKeyPair = RandomKeyPairAdapter()
+            issuerKeyPair = EphemeralKeyWithSelfSignedCert()
```

Comment:
I would not, in every test case, create a self-signed certificate, as it's not needed for most use cases

Reply:
does it impact the tests?

Reply:
Maybe, maybe not, I just would not change it, since certificates are needed only for some specific use cases

Reply:
done